docs/installation/autoware/source-installation.md

```diff
@@ -31,35 +31,35 @@ sudo apt-get -y install git
    cd autoware
    ```
 
-2. You can install the dependencies either manually or using the provided Ansible script.
+2. You can install the dependencies automatically using the provided Ansible script
```

Comment:
```suggestion
2. If you are newly installing Autwoare, you can install the dependencies automatically using the provided Ansible script
```

Reply:
Thank you for your suggestion. I merged this.